frontend/common/src/fakeData/fakePools.ts

```diff
@@ -36,6 +37,11 @@ const generatePool = (
     },
     classifications:
       faker.helpers.arrayElements<Classification>(classifications),
+    keyTasks: {
+      en: `EN ${faker.lorem.paragraph()}`,
+      fr: `FR ${faker.lorem.paragraph()}`,
+    },
+    status: faker.helpers.arrayElement<PoolStatus>(Object.values(PoolStatus)),
```

Comment:
Might as well add this to the PoolStories too, in stories/pool.stories.tsx

Reply:
Done 👍